stable/cluster-overprovisioner/README.md

```diff
@@ -34,24 +34,32 @@ Some thought or experimentation is required to set `deployment.resources` and `d
-| `nodeSelector`                     | Node labels for pod assignment                    | `{}`              |
-| `annotations`                      | Optional deployment annotations                   | `{}`              |
-| `fullnameOverride`                 | Override the fullname of the chart                | `nil`             |
-| `nameOverride`                     | Override the name of the chart                    | `nil`             |
-| `tolerations`                      | Optional deployment tolerations                   | `[]`              |
+| Parameter                                      | Description                                       | Default           |
+| -----------------------------------------------|---------------------------------------------------|-------------------|
+| `priorityClassOverprovision.name`              | Name of the overprovision priorityClass           | `overprovision`   |
+| `priorityClassOverprovision.value`             | Priority value of the overprovision priorityClass | `-1`              |
+| `priorityClassDefault.enabled`                 | If true, enable default priorityClass             | `true`            |
+| `priorityClassDefault.name`                    | Name of the default priorityClass                 | `default`         |
+| `priorityClassDefault.value`                   | Priority value of the default priorityClass       | `0`               |
+| `replicaCount`                                 | Number of replicas                                | `1`               |
+| `resources`                                    | Resources for the overprovision pods              | `{}`              |
+| `image.repository`                             | Image repository                                  | `k8s.gcr.io/pause`|
+| `image.tag`                                    | Image tag                                         | `3.1`             |
+| `image.pullPolicy`                             | Container pull policy                             | `IfNotPresent`    |
+| `affinity`                                     | Map of node/pod affinities                        | `{}`              |
+| `nodeSelector`                                 | Node labels for pod assignment                    | `{}`              |
+| `annotations`                                  | Optional deployment annotations                   | `{}`              |
+| `fullnameOverride`                             | Override the fullname of the chart                | `nil`             |
+| `nameOverride`                                 | Override the name of the chart                    | `nil`             |
+| `tolerations`                                  | Optional deployment tolerations                   | `[]`              |
+| `additionalDeployments`                        | Define optional additional deployments            | `[]`              |
```

Comment:
IMO, this is messy.

Just have list in `deployments` OR as it is now. But not `resources/replicaCount` AND also `additionalDeployments`.

Reply:
I don't get your point here.

I add `additionalDeployments`, related to not break old stuff for existing user. (it's additional)
For my additional deployments, i have every time different requirements for `nodepool/resources/replicaCount`, so I need to configure them separate.

Reply:
You don't need to worry about breaking stuff as the charts are versioned. An existing user will not have a problem unless they update their chart version. Any Helm users should know to pin their charts to a specific version for this very reason.

> For my additional deployments, i have every time different requirements for nodepool/resources/replicaCount, so I need to configure them separate.

Yes, I get the point, but it would have been cleaner to just move to your method of looping over a list of deployments and drop the old way, rather than support both.

Doesn't matter, it's merged now anyway 😃

Reply:
okay, yes go point. the next time i will think on this.

Reply:
We are still on time to create a new PR

Reply:
created a new PR under https://github.com/helm/charts/pull/16686 to cleanup old deployment style.